### PR TITLE
[Teams] add self-diagnostics section + warp cleaning

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

File: products/cloudflare-one/src/content/connections/connect-devices/warp/deployment/macOS-Teams.md [hunk 1]
```diff
@@ -121,7 +121,10 @@ If your organization uses Teams Access policies to control device registration,
 1. Navigate to **Preferences**.
 1. Select the **Account** tab.
 1. Click **Login with Cloudflare for Teams**.
+
+ ![Preferences UI in the Account tab with the Login to Teams button circled](../../../../static/documentation/connections/macOS_TeamsButton.png)
+
 1. Enter your organization name (if your auth domain were `https://example.cloudflareaccess.com`, you would enter `example`).
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/deployment/macOS-Teams.md]: The numbering restarts here. Small nit

File: products/cloudflare-one/src/content/connections/connect-devices/warp/device-enrollment.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 order: 3
 ---
 
-# Device enrollment
+# Device enrollment policies
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/device-enrollment.md:5]: Idk if I agree with calling them policies in this context given it's just an initial setup vs. defining a policy

File: products/cloudflare-one/src/content/connections/connect-devices/warp/install-cloudflare-cert.md [hunk 3]
```diff
@@ -38,27 +38,31 @@ You will need to install the root certificate in the Keychain Access application
...
 
 1. Download the Cloudflare certificate [here](../../../static/documentation/connections/Cloudflare_CA.crt).
 
 2. Double-click on the `.crt` file.
 
-  The certificate is now listed in the **Keychain Access** application, under the **login** keychain. If you want to install it in the **System** keychain instead, drag and drop the certificate from the login keychain onto the System keychain section on the left.
+3. In the pop-up message, choose the option that suits your needs (*login*, *Local Items*, or *System*) and click **Add**.
 
-![Keychain](../../../static/documentation/connections/listed-in-keych.png)
+ ![Popup](../../../static/documentation/connections/keychain-popup.png)
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/install-cloudflare-cert.md:49]: This looks like a broken image

File: products/cloudflare-one/src/content/faq/self-diagnostics.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+The [Teams help page](https://help.teams.cloudflare.com/) at `https://help.teams.cloudflare.com/` is a quick way to check whether Gateway is successfully protecting your network. It also allows your users to further investigate network issues.
+
+This page shows:
+
+* An overview of **Connectivity** details
+* **Debug information** 
+* An overview of the content categories and security risks that are being blocked
+
+![Help page](../static/documentation/faq/help-page.png)
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/faq/self-diagnostics.md:34]: This looks like another broken image

File: products/cloudflare-one/src/content/connections/connect-devices/warp/install-cloudflare-cert.md (hunk 3 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/connections/connect-devices/warp/install-cloudflare-cert.md:49]: weird, all these images build locally

**Review by kennyj42 — CHANGES_REQUESTED** [review]
Just a small tweaks required

File: products/cloudflare-one/src/content/connections/connect-devices/warp/install-cloudflare-cert.md (hunk 3 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/connections/connect-devices/warp/install-cloudflare-cert.md:49]: i'd say let's merge and see how it looks in production

**Review by kennyj42 — APPROVED** [review]

**Pull request merged**